### DISCUSSION TITLE
1.4.0rc1 released

### DISCUSSION THREAD

**Discussion by gustavo-iniguez-goya** [evilsocket/opensnitch#391] [General]
New version released: https://github.com/evilsocket/opensnitch/releases/tag/v1.4.0-rc.1

It's marked as a pre-release, meaning that there could be some bugs (specially when loading/creating rules). However as far as I can tell, it works fine.

Please, install it and let us know if you find any problems.

**Comment by raboof** [comment]
It installs fine for me (https://github.com/NixOS/nixpkgs/pull/118329), but I had to switch the default action in the daemon configuration to `deny` to get back to my desired behavior as discussed in https://github.com/evilsocket/opensnitch/issues/392#issuecomment-812565300 .

**Comment by gustavo-iniguez-goya** [comment]
Regarding the lists of domains, while we implement the scheduled updates, you can use this simple script to update every n hours/days the lists of ads/spam/etc:
[update_adlists.sh.txt](https://github.com/evilsocket/opensnitch/files/6251697/update_adlists.sh.txt)


howto:
- mkdir ~/ads/
- mv ~/Downloads/update_adlists.sh.txt ~/ads/update_adlists.sh
- chmod +x ~/ads/update_adlists.sh
- crontab -e
- 0 15 * * * /home/<YOUR_USER>/ads/update_adlists.sh

related: https://github.com/evilsocket/opensnitch/issues/298#issuecomment-785878727 , #298 
this also deprecates the old script: https://github.com/evilsocket/opensnitch/blob/master/make_ads_rules.py
